docs/ocaml/static/css/odoc.css

```
@@ -747,6 +747,26 @@ h1+.modules, h1+.sel {
   text-decoration: underline;
 }
 
+/*!* Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome* License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)*/
+@font-face{
+	font-family:FontAwesome;
+	src:url(../../../static/css/fonts/fontawesome-webfont.eot?674f50d287a8c48dc19ba404d20fe713);
+	src:url(../../../static/css/fonts/fontawesome-webfont.eot?674f50d287a8c48dc19ba404d20fe713?#iefix&v=4.7.0) format("embedded-opentype"),url(../../../static/css/fonts/fontawesome-webfont.woff2?af7ae505a9eed503f8b8e6982036873e) format("woff2"),url(../../../static/css/fonts/fontawesome-webfont.woff?fee66e712a8a08eef5805a46892932ad) format("woff"),url(../../../static/css/fonts/fontawesome-webfont.ttf?b06871f281fee6b241d60582ae9369b9) format("truetype"),url(../../../static/css/fonts/fontawesome-webfont.svg?912ec66d7572ff821749319396470bde#fontawesomeregular) format("svg");
+	font-weight:400;
+	font-style:normal
+}
+.fa,.icon,.rst-content .admonition-title,.rst-content .code-block-caption .headerlink,.rst-content code.download span:first-child,.rst-content dl dt .headerlink,.rst-content h1 .headerlink,.rst-content h2 .headerlink,.rst-content h3 .headerlink,.rst-content h4 .headerlink,.rst-content h5 .headerlink,.rst-content h6 .headerlink,.rst-content p.caption .headerlink,.rst-content table>caption .headerlink,.rst-content tt.download span:first-child,.wy-menu-vertical li.current>a span.toctree-expand,.wy-menu-vertical li.on a span.toctree-expand,.wy-menu-vertical li span.toctree-expand{
```

```suggestion
.icon, .icon:focus {
```
This seems to be sufficient and also fixes the issue with the icon not being found when in focus (when clicking the icon).